storybook/stories/API/props/AnimationProps.ts

```diff
@@ -9,24 +9,37 @@ export const animationBegin = {
   description: 'Specifies when the animation should begin, the unit of this option is ms.',
   type: { name: 'number' },
   defaultValue: 0,
```

Comment:
Now the defaultValue seems to be defined twice. Is that necessary?

Reply:
Hi @nikolasrieble, without the `defaultValue` in the `table` prop, the default value in the args table is not set to the manually set `defaultValue` of the argType, but only to the inferred default value. 

According to https://github.com/storybookjs/storybook/issues/11983#issuecomment-896790639 (which also refers to https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#no-longer-inferring-default-values-of-args):

> To affect the Default column in the ArgsTable, you should set the argTypes.<field>.table.defaultValue field. The value should be an object with the keys summary and optionally details, which are both strings.